.meta/_partials/_encoding.toml

```diff
@@ -0,0 +1,23 @@
+[<%= namespace %>.encoding.format]
```

Comment:
Ideally we'd define an enumeration for this. What if we passed an `encodings` variable to this partial. Ex:

```
 <%= render("_partials/_encoding.toml", namespace: "sinks.aws_cloudwatch_logs.options", encodings: ["json", "text"]) %>
```

Then we could use `if` statements to define the enumerations:

```toml
[<%= namespace %>.encoding.format.enum]
<%- if encodings.include?("text") -%>text = "..."<%- end -%>
<%- if encodings.include?("json") -%>json = "..."<%- end -%>
<%- if encodings.include?("ndjson") -%>text = "..."<%- end -%>
```